src/QuickMarcEditor/useSaveRecord/useSaveRecord.test.js

```diff
@@ -34,6 +35,11 @@ jest.mock('react-router-dom', () => ({
   useParams: jest.fn(() => ({})),
 }));
 
+jest.mock('@folio/stripes/core', () => ({
+  ...jest.requireActual('@folio/stripes/core'),
+  checkIfUserInCentralTenant: jest.fn().mockReturnValue(false),
+}));
```

Comment:
The `checkIfUserInCentralTenant` is already mocked in the `test` folder. This can be removed.